### ISSUE TITLE
Allow use of UI changes while game is busy

### ISSUE THREAD

**Issue by Krwak** [cataclysmbnteam/Cataclysm-BN#2941]
### Is your feature request related to a problem? Please describe.

Not sure how feasible this would be but putting it here cause it would be nice to have:
When game is busy sleeping or crafting or reading, it would be great to be able to see stats of a character or toggle minimap or check inventory etc.
This would give player something to do while waiting for the game to finish a process that it's doing.

### Describe the solution you'd like

Basically allow interacting with UI while the game is processing things.

**Comment by AngelicosPhosphoros** [comment]
Well, it is not easy to do because game's UI code is mixed with update code.
